src/gluonts/evaluation/_base.py

```diff
@@ -256,6 +257,10 @@ def __call__(
             desc="Running evaluation",
         ) as it, np.errstate(divide="ignore", invalid="ignore"):
             if self.num_workers and not sys.platform == "win32":
+
+                if torch.cuda.is_available():
+                    torch.multiprocessing.set_start_method('spawn', force=True)
+
```

Comment:
I think this should not happen here, it's probably better to keep it on the caller side

Reply:
For now I put it into `src/gluonts/torch/__init__.py`, what do you think of it?

Reply:
Yes, maybe part of the test is better